openvdb/openvdb/math/Vec3.h

```diff
@@ -668,6 +669,21 @@ OPENVDB_IS_POD(Vec3ui)
 OPENVDB_IS_POD(Vec3s)
 OPENVDB_IS_POD(Vec3d)
 
+// Half WIP
+template<>
+inline auto cwiseAdd(const Vec3<math::internal::half>& v, const float s)
```

Comment:
You're right. Thanks for catching this, Dan.

Reply:
Just pushed an updated version of this PR that moves this function to `Types.h` because that's where we first define `math::half` depending on the macro `OPENVDB_USE_IMATH_HALF`.